### PR TITLE
Updates tenant upgrade instructions with prereqs

### LINKED ISSUE #963: [K8s] Update tenant upgrade guide with prereqs
The Github tenant upgrade docs mention required prereqs for tenants that if not done can lead to problems with volume claims.

Update the tenant upgrade docs to reflect the prerequisites listed at https://github.com/minio/operator/blob/master/docs/upgrade.md.

Might as well verify other changes aren't needed to keep parity with Github instructions.

### PR DESCRIPTION
**Pull request by djwfyi**
Community member noted that the prereqs were out of date in our tenant upgrade docs.
This brings them in line with what's in Github.

Closes #963

Staged
- http://192.241.195.202:9000/staging/tenant-upgrade/k8s/operations/install-deploy-manage/upgrade-minio-tenant.html

### CODE REVIEW

File: source/operations/install-deploy-manage/upgrade-minio-tenant.rst [hunk 1]
```diff
@@ -17,12 +17,126 @@ MinIO Kubernetes Operator and Plugin
...
+This procedure assumes the latest stable Operator and Plugin, version |operator-version-stable|.
 
 See :ref:`deploy-operator-kubernetes` for complete documentation on deploying the MinIO Operator.
 
+Install the Plugin
+++++++++++++++++++
+
 .. include:: /includes/k8s/install-minio-kubectl-plugin.rst
 
+Upgrading from Operator v4.2.2
```
Comment (ravindk89) [inline source/operations/install-deploy-manage/upgrade-minio-tenant.rst:29]: Can we add a rellink above and reference this in a step from the `upgrade-minio-operator` pages too?

**Review by ravindk89 — CHANGES_REQUESTED** [review]
One request for scope creep - should reference this very good information in the Upgrade Operator pages.

File: source/operations/install-deploy-manage/upgrade-minio-tenant.rst (hunk 1 above)
Comment (djwfyi) [reply source/operations/install-deploy-manage/upgrade-minio-tenant.rst:29]: Not needed. This information is already on that page.
There's duplication involved here, but I don't see the need to clean it up in this PR.

**Review by ravindk89 — APPROVED** [review]

**Review by feorlen — APPROVED** [review]
LGTM

**Pull request merged**